changelog.md

```diff
@@ -1,12 +1,18 @@
 # Changelog
 
+# NEXT
+
+- Improved the topological order to take into account time.
```

Comment:
"Improved the `cai_causal_graph.time_series_causal_graph.TimeSeriesGraph.get_topological_order` method for a `cai_causal_graph.time_series_causal_graph.TimeSeriesGraph` to better account for time."